src/domain/opendatafabric/src/identity/dataset_identity.rs

```diff
@@ -257,7 +268,7 @@ impl DatasetAliasPattern {
     }
 
     pub fn matches(&self, dataset_handle: &DatasetHandle) -> bool {
-        (self.account_name.is_none() || self.account_name == dataset_handle.alias.account_name)
+        self.account_name == dataset_handle.alias.account_name
```

Comment:
This seems incorrect. Shouldn't a pattern `foo` should match dataset with full alias `alice/foo`?

Reply:
True, return it back